### PR TITLE
Feat(eos_cli_config_gen): Add schema for access_lists

### PR DESCRIPTION
**Pull request by ClausHolbechArista**
### Contributor Checklist

- [x] Create schema fragment matching data model described in README.md and README_v4.0.md
  - README.md is most complete with all keys. README_v4.0 includes partial data models after conversion to lists.
  - Schema fragment path is `roles/eos_cli_config_gen/schemas/schema_fragments/<data_model_key>.schema.yml`.
  - Copy line 1-6 from another schema (comments and global schema).
  - Refer to [schema documentation](https://avd.sh/en/devel/docs/input-variable-validation-BETA.html) for syntax and/or use YAML Lint plugin from Redhat in VSCode.
  - Use `convert_types` on value that could be mixed type or misinterpreted like integers and numeric strings.
- [x] If the data model has been converted from wildcard dicts:
  - Add `convert_types: ['dict']` to the schema.
  - Remove `convert_dicts` from the `templates/eos/<>.j2` and `templates/documentation/<>.j2` templates.
- [x] Run `molecule converge -s build_schemas_and_docs` to update schema and documentation.
- [x] Test by running `molecule converge -s eos_cli_config_gen` and verify no errors or changes to generated configs/docs.

### Reviewer Checklist

- Reviewer 1:
  - [x] Verify that data model is fully covered in the described schema. Easiest by looking at the generated documentation.
  - [x] Verify that `convert_dicts` has been removed from templates as applicable.
  - [x] Verify no changes to configs/docs on any molecule scenario
  - [x] Verify that CI pass

- Reviewer 2:
  - [ ] Verify that data model is fully covered in the described schema. Easiest by looking at the generated documentation.
  - [ ] Verify that `convert_dicts` has been removed from templates as applicable.
  - [ ] Verify no changes to configs/docs on any molecule scenario
  - [ ] Verify that CI pass

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml [hunk 1]
```diff
@@ -1 +1,55 @@
...
+    - dict
+    display_name: IP Extended Access-Lists
+    description: 'AVD currently supports 2 different data models for extended ACLs:
+
+
+      - The legacy `access_lists` data model, for compatibility with existing deployments
+
+      - The improved `ip_access_lists` data model, for access to more EOS features
+
+
```
Comment (gmuloc) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml:17]: Extra lines here as compared to fragment

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/access_lists.schema.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      Access list names must be unique.
+
+      The legacy data model supports simplified ACL definition with `sequence_number` to `action_string` mapping:
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          required: true
+          display_name: access_list_name
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/access_lists.schema.yml]: Better display name needed. Same elsewhere

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml (hunk 1 above)
Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml:17]: Notice the difference in line 10 between ' and |. The yaml formatting for newlines inside strings is just to have two of them... So this is correct.

Comment (gmuloc) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml:17]: indeed - my bad

**Review by KonikaChaurasiya-GSLab — APPROVED** [review]
LGTM!

**Review by gmuloc — APPROVED** [review]
LGTM

**Pull request merged**